include/aktualizr-lite/cli/cli.h

```diff
@@ -34,12 +34,30 @@ enum class StatusCode {
   InstallRollbackFailed = 130,
 };
 
+/**
+ * The installation mode to be applied. Specified during InstallContext context initialization.
+ */
+enum class PullMode {
+  /**
+   * Default mode, do pull target during install operation.
+   */
+  PerformPull = 0,
+  /**
+   * Do no pull target during install. Target is expected to be pulled before.
+   */
+  SkipPull,
+};
+
 StatusCode CheckIn(AkliteClient &client, const LocalUpdateSource *local_update_source);
 
+StatusCode Pull(AkliteClient &client, int version = -1, const std::string &target_name = "",
+                InstallMode install_mode = InstallMode::All, bool force_downgrade = true,
```

Comment:
Do we really need `install_mode` for `Pull`?

Reply:
The value is passed during the Installer instantiation, and the internal Download has access to it. But it is not used, and sounds like there would be no reason to use it there. I'll remove the parameter.

Reply:
Removed `install_mode` from `Pull`